src/ape/api/networks.py

```diff
@@ -605,6 +605,18 @@ def config(self) -> PluginConfig:
     def _network_config(self) -> Dict:
         return self.config.get(self.name, {})
 
+    @cached_property
+    def gas_limit(self) -> Union[str, int]:
```

Comment:
You could use the `Union[Literal["auto"], Literal["max"], int]` type

Reply:
Sweet! Added in ecd6d75d